crowdloans/config/polkadot.json

```diff
@@ -0,0 +1,14 @@
+{
+  "NAME": "polkadot",
+  "ASSET_ID": 101,
+  "UNIT": "DOT",
+  "SS58_FORMAT": 0,
+  "PARA_BONUS_UNIT": "PARA",
+  "PARALLEL_PARA_ID": "2012",
+  "HIDDEN_PARAIDS": [],
+  "DISABLED_CONTRIBUTION_PARAIDS": ["2000", "2004", "2006"],
```

Comment:
Should we move this close status into each project?

Reply:
ok